keps/sig-release/3031-signing-release-artifacts/README.md

```diff
@@ -150,6 +154,272 @@ KEP focuses more on the "What" aspects rather than the "How".
 - All Kubernetes artifacts are signed. This does exclude everything which gets
   build outside of the main Kubernetes repository.
 
+## Production Readiness Review Questionnaire
```

Comment:
This is out of tree so PRR is optional. I'll quickly look through your questionnaire.